user_guide_src/source/installation/upgrade_pagination.rst

```diff
@@ -0,0 +1,61 @@
+Upgrade Pagination
+##################
+
+.. contents::
+    :local:
+    :depth: 1
+
+
+Documentations
+==============
+
+- `Pagination Class Documentation Codeigniter 3.X <http://codeigniter.com/userguide3/libraries/pagination.html>`_
+- :doc:`Pagination Documentation Codeigniter 4.X </libraries/pagination>`
+
+
+What has been changed
+=====================
+- In CI4 the pagination is a built-in method in the Model class. You have to change the views and also the controller in order to use the new pagination library.
+
```

Comment:
Other notable changes:
- CI4 pagination uses the actual page number only. You can't use the starting index (offset) for the items which is the default in CI3. 
- If you want to customize the pagination links, you need to create View Templates <https://codeigniter4.github.io/CodeIgniter4/libraries/pagination.html#customizing-the-links>.